src/components/Footer.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled, { css } from 'styled-components';
+import { ReactComponent as FooterLogo } from '../assets/sign.svg';
+
+const FooterWrapper = styled.footer`
+  display: flex;
+  align-items: center;
+  position: absolute;
+  bottom: 0;
+  height: 100px;
+  width: 100%;
+`;
+
+const Nav = styled.nav`
+  display: flex;
+  width: 100%;
+  margin: auto;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 250px;
+`;
+
+const baseLinkStyle = css`
+  color: ${({ theme }) => theme.color.navigation};
+  text-decoration: none;
+  font-family: ${({ theme }) => theme.font.primary};
+  font-size: 0.9rem;
```

Comment:
This results in a font-size of 14.4px which is slightly bigger than the one in the design (14px). That's why the styles don't match the design pixel-perfectly

Reply:
Btw: you can move this to the theme as well and call it something like `fontSizeSmall` or whatever you like